apiserver/src/main/java/org/dependencytrack/persistence/ProjectQueryManager.java

```diff
@@ -748,7 +746,7 @@ public void bind(final Project project, final Collection<Tag> tags) {
     public boolean hasAccess(final Principal principal, final Project project) {
         if (!isEnabled(ConfigPropertyConstants.ACCESS_MANAGEMENT_ACL_ENABLED)
                 || principal == null // System request (e.g. MetricsUpdateTask, etc) where there isn't a principal
-                || super.hasAccessManagementPermission(principal))
+                || (request != null && request.getEffectivePermissions().contains(Permissions.Constants.ALL_PROJECT_ACCESS)))
```

Comment:
NB: Checking the permission via `request.getEffectivePermissions()` does not require additional database operations like `super.hasAccessManagementPermission(principal)` did. However `request.getEffectivePermissions()` is only populated by `AuthorizationFilter`, which was not registered in a few resource tests before.